src/handlers/account-deletion-processor-handler.ts

```diff
@@ -55,8 +55,6 @@ function getUserId(record: SQSRecord) {
 async function updateDeleteStatusId(userId: string) {
```

Comment:
Just as a general comment on this file, could we possibly get code documentation here?